x-pack/platform/plugins/shared/onechat/server/services/conversation/client/converters.ts

```diff
@@ -139,7 +141,8 @@ export const updateConversation = ({
     ...conversation,
     ...update,
     space,
-    updatedAt: updateDate.toISOString(),
+    updated_at: updateDate.toISOString(),
+    attachments: update.attachments ?? conversation.attachments,
```

Comment:
Isn't that unnecessary with the base `{ ...conv, ...update }` spread? But if attachments default to an empty array, maybe we need to check that?